app/views/admin/data_update_scripts/index.html.erb

```diff
@@ -7,23 +7,66 @@
         <th scope="col" class="whitespace-nowrap">Status</th>
         <th scope="col">Created at</th>
         <th scope="col">Run at</th>
+        <th scope="col">Re-run</th>
       </tr>
     </thead>
     <tbody class="crayons-card">
       <% @data_update_scripts.each do |script| %>
-        <tr>
-          <td class="whitespace-nowrap"><%= script.id %></td>
-          <td><%= script.file_name %></td>
-          <td class="justify-content-center">
+        <tr class="<%= 'alert-danger' if script.status == 'failed' %>" id="data_update_script_<%= script.id %>_row">
+          <td id="data_update_script_<%= script.id %>" class="whitespace-nowrap"><%= script.id %></td>
+          <td id="data_update_script_<%= script.id %>_file_name"><%= script.file_name %></td>
+          <td id="data_update_script_<%= script.id %>_status" class="justify-content-center">
             <%= script.status %>
             <% if script.error.present? %>
               <div class="fs-xs"><%= script.error %></div>
             <% end %>
           </td>
-          <td><%= script.created_at %></td>
-          <td><%= script.run_at %></td>
+          <td id="data_update_script_<%= script.id %>_created_at"><%= script.created_at %></td>
+          <td id="data_update_script_<%= script.id %>_run_at" class="whitespace-nowrap"><%= script.run_at %></td>
+          <td id="data_update_script_<%= script.id %>_button" class="whitespace-nowrap">
+            <button onclick="forceRun(<%= script.id %>);return false;" type="button" className="crayons-btn crayons-btn--secondary">
+              Re-run
+            </button>
+          </td>
         </tr>
       <% end %>
     </tbody>
   </table>
 </main>
+
+<script>
+  function forceRun(id) {
+    let formData = new FormData();
+    formData.append('id', id);
+
+    let statusColumn = document.getElementById(`data_update_script_${id}_status`);
+    let runAtColumn = document.getElementById(`data_update_script_${id}_run_at`);
+    let button = document.getElementById(`data_update_script_${id}_button`);
+
+    statusColumn.innerHTML = "loading..";
+    runAtColumn.innerHTML = "loading..";
+    button.innerHTML = "loading..";
+
+    fetch('/admin/run_data_update_scripts', {
+      method: 'POST',
+      headers: {
+       'X-CSRF-Token': document.querySelector("meta[name='csrf-token']").content,
+      },
+      body: formData,
+      credentials: 'same-origin'
+    }).then(response => response.json()
+      .then(json => {
+
+        let updatedScript = json.response;
+        // escape out the <>
```

Comment:
To fix: Will remove this comment once I fix this issue. The html in the error is being rendered as html instead of being shown as a string.